build.gradle

```diff
@@ -84,16 +84,16 @@ class GradlePluginMetaData {
 
 def pluginList = [
     new GradlePluginMetaData(
-        "${project.name}-base",
+        'latte-base',
```

Comment:
We have 'project.name' defined in settings.gradle file. Not sure why are we using hardcoded name as 'latte' here and other places in build.gradle.

Reply:
I unwound this because I couldn't get it to compile otherwise with the package name, project name, group name spread in various placees ..it actually didn't even build when I first pulled it down and I got frustrated.

I'll revert the project name ones.  I don't having a project name as a substitute serves any purpose.  is left over from original fork.

Reply:
i've added project name back in...but for package names I left them since they didn't match exactly with group name.